src/components/Searchbar/Searchbar.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import ResultField from "./ResultField";
+import CloseButton from "./CloseButton";
+import atractions from "./dummyData"; // podmień na dumm data z projektu kiedy powstanie
+import "./searchbar.css";
+
+const Searchbar = () => {
+  const [searchInput, setSearchInput] = useState("");
+  const [atractionList, setatractionList] = useState([]);
+
+  const handleChange = (e) => {
+    setSearchInput(e.target.value);
+    setatractionList(atractions);
+  };
+
+  const clearInput = (e) => {
+    setSearchInput("");
+  };
+
+  return (
+    <>
```

Comment:
Here React.Fragment isn't necessary either, we only have one element - a div which wraps everything up.

Reply:
ok:)